curations/nuget/nuget/-/Microsoft.Data.SqlClient.SNI.runtime.yaml

```diff
@@ -9,3 +9,6 @@ revisions:
   2.1.1:
     licensed:
       declared: OTHER
+  5.1.0:
+    licensed:
+      declared: MIT
```

Comment:
GitHub repo linked by nuget package is MIT, yet NuGet has unnamed license type text included with nuget. Not sure which it is. but this is not properly detect by Component Governance.